tests/wallet/cat_wallet/test_trades.py

```diff
@@ -329,20 +307,14 @@ async def test_cat_trades(
                         "xch": {"set_remainder": True},
                         "cat": {"init": True, "set_remainder": True},
                     },
-                    post_block_balance_updates={
-                        "xch": {"set_remainder": True},
-                        "cat": {"set_remainder": True},
-                    },
+                    post_block_balance_updates={"xch": {"set_remainder": True}, "cat": {"set_remainder": True}},
                 ),
                 WalletStateTransition(
                     pre_block_balance_updates={
                         "xch": {"set_remainder": True},
                         "new cat": {"init": True, "set_remainder": True},
                     },
-                    post_block_balance_updates={
-                        "xch": {"set_remainder": True},
-                        "new cat": {"set_remainder": True},
-                    },
+                    post_block_balance_updates={"xch": {"set_remainder": True}, "new cat": {"set_remainder": True}},
```

Comment:
it's not obvious to me that a line that's so long github has to wrap it is better than two lines

Reply:
github doesn't wrap this for me and we have our existing maximum line width enforcement by black so this clearly fits under that limit.